cli/provisionercreate.go

```diff
@@ -0,0 +1,47 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/xerrors"
+
+	"github.com/coder/coder/codersdk"
+)
+
+func provisionerCreate() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create [name]",
+		Short: "Create a provisioner daemon instance",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			client, err := CreateClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			provisionerName := args[0]
+
+			provisionerDaemon, err := client.CreateProvisionerDaemon(cmd.Context(), codersdk.CreateProvisionerDaemonRequest{
+				Name: provisionerName,
+			})
+			if err != nil {
+				return err
+			}
+
+			if provisionerDaemon.AuthToken == nil {
```

Comment:
Kind of related to my other question about nullability of the `auth_token` field, but why would this ever be allowed to happen during `create`?

Reply:
Yeah, it shouldn't be possible. A null `auth_token` would indicate that the provisioner was incorrectly registered as "in-process" rather than "out-of-process". But if that does somehow happen, this is just a sanity check so that we generate a meaningful error message rather than panicking.

Reply:
Could we make it so the API errors instead (and doesn't return a nullable UUID)? (I think that would be nicer for consumers in general.)